### PR TITLE
Numpy: bump

### PR DESCRIPTION
**Pull request by extrowerk**
Depends on https://github.com/haikuports/haikuports/pull/5466

### CODE REVIEW

**Comment by Begasus** [comment]
For this there has to be done some changes (cmd:cython$pythonVersion$secondaryArchSuffix) (provided the buildbot builds the ok) :)

**Comment by extrowerk** [comment]
@Begasus: any hint what should be changed?

File: dev-python/numpy/numpy-1.19.4.recipe [hunk 1]
```diff
@@ -56,13 +56,14 @@ eval "PROVIDES_${pythonPackage}=\"\
 	\"; \
 REQUIRES_$pythonPackage=\"\
 	haiku$secondaryArchSuffix\n\
-	numpy$secondaryArchSuffix\n\
+	cmd:cython$pythonVersion\n\
```
Comment (Begasus) [inline dev-python/numpy/numpy-1.19.4.recipe:59]: The cython (for python 3.6 and 3.7) packages for 32 bit are secondary architecture so (same for BUILD_PREREQUIRES):
```diff
-	cmd:cython$pythonVersion\n\
+	cmd:cython$pythonVersion$secondaryArchSuffix\n\
```

**Comment by Begasus** [comment]
Maybe wait until https://github.com/haikuports/haikuports/pull/5447 is merged?

**Comment by extrowerk** [comment]
dependency fixed, thanks

**Pull request merged**